robolectric/src/test/java/org/robolectric/shadows/ShadowDisplayManagerGlobalTest.java

```diff
@@ -123,7 +123,41 @@ private VirtualDisplay createVirtualDisplay(Surface surface) {
             .setSurface(surface)
```

Comment:
The line of code that creates a `VirtualDisplayConfig` can only be run on SDK >= R.